commands/comment.go

```diff
@@ -58,7 +58,7 @@ func checkCommentLocation(repo repository.Repo, commit, file string, line uint)
 		return err
 	}
 	lines := strings.Split(contents, "\n")
-	if line >= uint(len(lines)) {
+	if line < 1 || line > uint(len(lines)) {
```

Comment:
@ojarjur Sorry, I don't understand what you mean. -l can be taken from 1 to count of lines. So 1 to len(lines).